pulp_smash/config.py

```diff
@@ -352,13 +357,14 @@ class PulpSmashConfig(object):
         http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
     """
 
-    def __init__(self, pulp_auth=None, pulp_version=None, systems=None):
+    def __init__(self, pulp_auth=None, pulp_version=None, systems=None, selinux_enabled=None):
```

Comment:
All of these values default to None. But the pydoc for the class suggests they default to useful values. The current default-enforcing behavior is actually in `.read()`

Reply:
The strategy adopted here is good. The weird discrepancy between documented constructor behaviour and actual constructor behaviour can be fixed at a later time. See: https://github.com/PulpQE/pulp-smash/issues/981